spec/models/solr_document_spec.rb

```diff
@@ -26,35 +26,39 @@
     end
   end
 
-  describe '#publishable?' do
```

Comment:
Is there a reason you removed this test?

Reply:
@jcoyne rebase mishap. I'll fix it in the morning. Good 👁️

Reply:
Restored.